### DISCUSSION TITLE
[A-L] (20 Giugno 2024) Esercizio 5

### DISCUSSION THREAD

**Discussion by FeddyLix17** [sapienzastudentsnetwork/metodi-matematici-per-l-informatica#415] [Esami A-L]
Mario, Claudia e Gianni hanno fatto un test di ammissione. Rilasciano le seguenti dichiarazioni.

- Mario dice: "Se io ho superato il test e Gianni non lo ha superato allora neanche Claudia lo ha superato."

- Claudia dice: "O Mario e Gianni superano entrambi il test o nessuno dei due lo ha superato."

- Gianni dice: "Se Mario ha superato il test allora Claudia non ha superato il test."

---

1. Formalizzare le tre affermazioni in logica proposizionale scegliendo un linguaggio adeguato. [2 punti]

2. È possibile che tutti e tre dicano il vero? Argomentare. [1.5 punti]

3. È possibile che tutti e tre dicano il falso e che abbia superato l’esame esattamente una persona? Argomentare. [1.5 punti]

(NB: le risposte vanno argomentate/dimostrate in base alla formalizzazione proposta, usando le nozioni rigorose di verità, conseguenza logica, assegnamento, soddisfazione, tavola di verità etc.).

**Comment by cek753** [comment]
<img width="585" height="544" alt="image" src="https://github.com/user-attachments/assets/f06673c7-1fa7-4a35-9058-3f48fad8f2fc" />
